### PR TITLE
Create 02.md

### PR DESCRIPTION
**Pull request by Superman-777**

### CODE REVIEW

File: 01/01/02.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Privet
+
+Cat and dog
```
Comment (Superman-777) [inline 01/01/02.md:3]: This is a mouse. Not the cat.

**Review by Superman-777 — COMMENTED** [review]
see notes

File: 01/01/02.md (hunk 1 above)
Comment (Victoria-Fire) [inline 01/01/02.md:3]: Not cat. This mouse.

**Review by Victoria-Fire — CHANGES_REQUESTED** [review]
See notes

File: 01/01/02.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# Privet
+
```
Comment (Victoria-Fire) [inline 01/01/02.md:2]: lgfm

**Review by Victoria-Fire — APPROVED** [review]

**Pull request merged**